CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this library adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - `Double?.convertUsdToZec` has been added as we are moving away from `BigDecimal` in favor of primitive types
 - `Locale.getDefault()` has been added
+- support for shielded transactions
```

Comment:
This changelog entry is incorrect (the SDK has supported shielded transactions since the beginning). Fix this to document whatever change is being made to the public API, or remove it if no public API changes are being made.